src/dmd/hdrgen.d

```diff
@@ -1227,8 +1227,9 @@ public:
     override void visit(LinkDeclaration d)
     {
         const(char)* p;
-        switch (d.linkage)
+        final switch (d.linkage)
         {
+        case LINK.default_:
```

Comment:
Where does this come from?

Reply:
Looks like case is in wrong place

Reply:
https://github.com/dlang/dmd/blob/1018db023831a8c348bb8f9121b08fabb6e32fc2/src/dmd/globals.d#L418-L428

Isn't `Link.default` equal to `LINK.d`?

Reply:
It should be considered as a no linkage, or unset.

Reply:
Imho the best solution would be to avoid having `LINK.default_` and `LINK.system` in the first place, but that's something for another PR..
-> I moved it to the position of the previous `default` case to avoid the PR being stalled. In other words: it doesn't change the status quo.